packages/testing/examples/useGasPrice.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { useGasPrice } from '@usedapp/core'
+import chai, { expect } from 'chai'
+import { solidity } from 'ethereum-waffle'
+import { renderWeb3Hook } from '../src'
+
+chai.use(solidity)
+
+describe('useGasPrice', () => {
+  it('retrieves gas price', async () => {
+    const { result, waitForCurrent } = await renderWeb3Hook(useGasPrice)
+    await waitForCurrent((val) => val !== undefined)
+
+    expect(result.error).to.be.undefined
+    expect(result.current.toNumber()).to.be.a('number')
```

Comment:
I think it should be like this:
```ts
expect(result?.current?.toNumber()).to.be.a('number')
```
Otherwise typescript will say that object is possibly undefined.

Reply:
Updated, `result` is expected not to be undefined therefore I used `result.current?.toNumber()`